gremlin-dotnet/src/Gremlin.Net/Gremlin.Net.csproj

```diff
@@ -36,8 +36,15 @@ Apache TinkerPop™ is a graph computing framework for both graph databases (OLT
 
 Gremlin.Net implements Gremlin within .NET. C# syntax has the same constructs as Java including "dot notation" for function chaining (a.b.c), round bracket function arguments (a(b,c)), and support for global namespaces (a(b()) vs a(__.b())). As such, anyone familiar with Gremlin-Java will immediately be able to work with Gremlin.Net. Moreover, there are a few added constructs to Gremlin.Net that make traversals a bit more succinct.
 
-Please see the reference documentation of Apache TinkerPop for more information on usage and use our Google Group gremlin-users if you have any questions:
-https://groups.google.com/forum/#!forum/gremlin-users
+Gremlin.Net is designed to connect to a "server" that is hosting a TinkerPop-enabled graph system. That "server" could be Gremlin Server
```

Comment:
You probably want to edit the template instead of this generated file directly.

Reply:
oh wow....thanks. i knew there was a reason i asked for review :)